packages/algob/sample-project/common/.gitignore

```diff
@@ -0,0 +1,6 @@
+/node_data
```

Comment:
Why you add node_date here? and why use add `/`(in root folder)?

Reply:
Because I think the node_data reference to data of node in algorand so we do not need to commit node_data

Reply:
Oh / mean from root project not root of OS, right?

Reply:
Yes that's right